composer.json

```diff
@@ -19,6 +19,8 @@
         "drupal/core-composer-scaffold": "^9.1",
         "drupal/core-project-message": "^9.1",
         "drupal/core-recommended": "^9.1",
+        "drupal/rules": "3.x-dev@dev",
```

Comment:
Why are these modules being installed?

Reply:
I have installed this module trying to send verification mail to user.